interfaces/builtin/process_control.go

```diff
@@ -60,6 +60,7 @@ setpriority
 sched_setaffinity
 sched_setparam
 sched_setscheduler
+scheduler_setattr
```

Comment:
I discussed this elsewhere and the syscall is fine to add to this interface. However the syscall is `sched_setattr` :) Also, since you are changing that, can you put it between sched_setaffinity and sched_setparam (for locally alphabetizing).

Reply:
Hah, I just renamed the pull request title. I didn't get to read the diff yet.

Question on the design: do we fell safe allowing `SCHED_RR`?

Reply:
ah haha sorry folks, let me force push the right one :-)